post-scan-actions/aws-python-conformity-custom-check/handler.py

```diff
@@ -27,8 +27,12 @@ def get_cc_accountid(awsaccountid):
     r = http.request("GET", accountsapi, headers=headers)
     accounts = json.loads(r.data.decode("utf-8"))["data"]
     for account in accounts:
-        if account["attributes"]["awsaccount-id"] == awsaccountid:
-            return account["id"]
+        try:
+            account["attributes"]["awsaccount-id"]
```

Comment:
Why would you need this line duplicated?

Reply:
Oops I thought I removed that already fixing now

Reply:
@carpusherw amended commit